src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -151,24 +151,110 @@
 short-summary: Manage vault private endpoint connections.
 """
 
+helps['keyvault private-endpoint-connection approve'] = """
```

Comment:
I'm a little surprised. Didn't we have help message for `approve` command before this PR?

Reply:
> I'm a little surprised. Didn't we have help message for `approve` command before this PR?

Yes we have, not in `help.py` but inherited from `custom.py`

Reply:
got it.